src/utilities/util_dump.c

```diff
@@ -994,7 +994,12 @@ dump_explore(WT_CURSOR *cursor, const char *uri, bool reverse, bool pretty, bool
 
     while (ret == 0) {
         i = num_args = 0;
-        fgets(user_input, MAX_ARGS, stdin);
+        if (fgets(user_input, MAX_ARGS, stdin) == NULL) {
+            if (!feof(stdin))
```

Comment:
```s
if (feof(stdin))
    goto err; /* Terminal is gone or user hit control-d. */
printf("Error: failed to read input.\n");
```
If `feof()` is true there is a reasonable chance the terminal is gone so printing out an error message serves no purpose.

Reply:
@marcbutler, as discussed, updated in 19dee4d8057ca35fe14c85f1d193788f03127ed2